File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex3.java
//Importando Scanner
import java.util.Scanner;

public class Ex3{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	System.out.println("Menor número\n\n");
	
	//vars
	float n1, n2;
	
	//Entradas
	System.out.print("Insira o 1° número: ");
	n1 = ent.nextFloat();
	
	System.out.print("Insira o 2° número: ");
	n2 = ent.nextFloat();
	
	//Processamento e Saída
	if ( n1 > n2 )
		System.out.println(n2 + " < " + n1);
	else if ( n2 > n1 )
		System.out.println(n1 + " < " + n2);
	else
		System.out.printf("%f = %f", n1, n2);
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex3.c
#include <stdio.h>

main()
{
    printf("Menor numero\n\n");

    //vars
    int n1, n2;


    //Entradas
    printf("Insira o 1o numero: ");
    scanf("%i", &n1);

    printf("Insira o 2o numero: ");
    scanf("%d",&n2);

    //Processamento e Saída
    if ( n1 == n2 )
        printf("Os numeros sao iguais\n");
    else if ( n1 < n2 )
        printf("%i eh menor que %d", n1, n2);
    else
        printf("%i eh menor que %d", n2, n1);


}
